{
	"deprecated": false,
	"cpeName": "cpe:2.3:a:percona:xtradb_cluster:8.0:*:*:*:*:*:*:*",
	"cpeNameId": "4D94DA48-6D19-479D-949B-7A8235A80176",
	"created": "2021-06-04T12:59:54.000",
	"lastModified": "2021-06-22T16:23:52.053",
	"titles": [
		{
			"title": "Percona XtraDB Cluster 8.0",
			"lang": "en"
		}
	],
	"refs": [
		{
			"ref": "https://www.percona.com/blog/2020/10/30/cve-2020-15180-affects-percona-xtradb-cluster/",
			"type": "Advisory"
		},
		{
			"ref": "https://github.com/percona/percona-xtradb-cluster/releases",
			"type": "Change Log"
		},
		{
			"ref": "https://bugzilla.redhat.com/show_bug.cgi?id=1894919",
			"type": "Advisory"
		}
	]
}
